flink-runtime/src/test/java/org/apache/flink/runtime/jobmaster/slotpool/SlotPoolImplTest.java

```diff
@@ -676,18 +586,18 @@ public void testFreeFailedSlots() throws Exception {
 	 */
 	@Test
 	public void testCreateAllocatedSlotReport() throws Exception {
+		final JobID jobId = new JobID();
 
-		try (SlotPoolImpl slotPool = createSlotPoolImpl()) {
+		try (SlotPoolImpl slotPool = createAndSetUpSlotPool(jobId)) {
 
 			final ArrayBlockingQueue<AllocationID> allocationIds = new ArrayBlockingQueue<>(1);
 			resourceManagerGateway.setRequestSlotConsumer(
-					slotRequest -> allocationIds.offer(slotRequest.getAllocationId()));
-
-			setupSlotPool(slotPool, resourceManagerGateway, mainThreadExecutor);
-			Scheduler scheduler = setupScheduler(slotPool, mainThreadExecutor);
+				slotRequest -> allocationIds.offer(slotRequest.getAllocationId()));
 
-			final SlotRequestId slotRequestId = new SlotRequestId();
-			final CompletableFuture<LogicalSlot> slotRequestFuture = allocateSlot(scheduler, slotRequestId);
+			final CompletableFuture<PhysicalSlot> slotRequestFuture = requestNewAllocatedSlot(
+				slotPool,
+				new SlotRequestId()
+			);
```

Comment:
this right bracket should be at the end of the previous line.

Reply:
Done.